keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -0,0 +1,916 @@
+
+###### Does enabling the feature change any default behavior?
+
+No, strict validation is false by default.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, not setting the query param will ensure we are not doing strict validation
+(i.e. disable the feature)
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+No harm, requests will resume performing strict validation
+
+###### Are there any tests for feature enablement/disablement?
+
+Testing for both presenence and absence of query param.
+
+### Rollout, Upgrade and Rollback Planning
+
+This section must be completed when targeting beta to a release.
+
+N/A
+
+<!--
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
+
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+
+###### What specific metrics should inform a rollback?
+
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+Even if applying deprecation policies, they may still surprise some users.
+-->
+### Monitoring Requirements
+
+This section must be completed when targeting beta to a release.
+
+N/A
+<!--
+###### How can an operator determine if the feature is in use by workloads?
+
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+
+###### How can someone using this feature know that it is working for their instance?
+
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+
+- [ ] Events
+  - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
```

Comment:
again, while not strictly required, it seems "cheap" for you to note the latency metrics and potentially the observed CPU used by the kube-apiserver.

Reply:
Noted latency/memory metrics in SLOs and monitoring in the SLIs section. Let me know if this is what you were expecting.